### PR TITLE
Chain and validator improvements

### PR DESCRIPTION
**Pull request by mpetrunic**
This is first in series of PRs to fix current priority bugs preventing chain finalization.

Bugs solved:
- block processing queue doesn't process block if you submit it before previous one is processed
- validator won't propose or attest before epoch 1
- cli displays genesis time in user friendly format instead of format required by cli

### CODE REVIEW

File: packages/lodestar-cli/src/commands/dev.ts [hunk 1]
```diff
@@ -136,8 +136,8 @@ export class DevCommand implements ICliCommand {
     if (options.genesisState) {
       state = quickStartOptionToState(config, tree, options.genesisState);
     } else if (options.genesisTime && options.validatorCount) {
-      logger.info(`Starting node with genesisTime ${new Date(parseInt(options.genesisTime)*1000)} and \
-       ${options.validatorCount} validators.`);
+      logger.info(`Starting node with genesisTime ${parseInt(options.genesisTime)} and `
```
Comment (wemeetagain) [inline packages/lodestar-cli/src/commands/dev.ts]: can we still print the int version of the genesis time?
like: ${new Date(...)} (${parseInt(...)})

File: packages/lodestar/src/chain/chain.ts [hunk 2]
```diff
@@ -411,4 +373,28 @@ export class BeaconChain extends (EventEmitter as { new(): ChainEventEmitter })
     this.logger.info(`Initializing beacon chain with eth1 block ${eth1Block.hash}`);
     await this.initializeBeaconChain(genesisState, merkleTree);
   };
+
```
Comment (wemeetagain) [inline packages/lodestar/src/chain/chain.ts:401]: Can you add a description/motivation of the algorithm/problem here?

**Review by wemeetagain — CHANGES_REQUESTED** [review]
Looks much better :+1:

File: packages/lodestar-cli/src/commands/dev.ts (hunk 1 above)
Comment (mpetrunic) [reply packages/lodestar-cli/src/commands/dev.ts]: You mean human readable version along with current int version?

File: packages/lodestar/src/chain/chain.ts (hunk 2 above)
Comment (mpetrunic) [reply packages/lodestar/src/chain/chain.ts:401]: sure

File: packages/lodestar-cli/src/commands/dev.ts (hunk 1 above)
Comment (wemeetagain) [reply packages/lodestar-cli/src/commands/dev.ts]: yea

**Review by wemeetagain — APPROVED** [review]
LGTM

**Pull request merged**